cursor.ts

```diff
@@ -139,6 +139,14 @@ export class Cursor extends Component {
     @property.enum(['input component', 'left', 'right', 'none'], 'input component')
     handedness: number | string = 0;
 
+    /** Object that has input component*/
+    @property.object()
+    inputObject: Object3D | null = null;
+
+    /** Object that has view component*/
```

Comment:
Your formatting doesn't seem properly configured.

```suggestion
    /** Object that has a view component */
```